spyder/config/base.py

```diff
@@ -536,17 +536,21 @@ def translate_gettext(x):
     MAC_APP_NAME = 'Spyder-Py2.app'
 
 
-def running_in_mac_app():
+def running_in_mac_app(pyexec=None):

```

Comment:
Since this is a simple PR, I think we should add the test here? I mena before fixing https://github.com/spyder-ide/spyder/pull/12271#pullrequestreview-408324323

Reply:
I don't think a test is required for this; it's behavior hasn't changed and just determines whether 'Spyder.app' is in the executable path.

Reply:
I agree, no problem leaving this untested (hard to test it, really).